test/runtests.jl

```diff
@@ -199,3 +201,23 @@ end
     @test_throws DimensionMismatch fit(KaplanMeier, [1, 2], [true])
     @test_throws ArgumentError fit(KaplanMeier, Float64[], Bool[])
 end
+
+@testset "Cox" begin
```

Comment:
Agreed! I've just noticed that, as `ModelFrame` always gives you matrices of `Float64` that's the only thing I've been testing, so I should probably make sure that other types behave well. What else do you have in mind?